src/pages/groups/List.js

```diff
@@ -187,6 +201,7 @@ export const GroupList = ajaxCaller(class GroupList extends Component {
           h(NewGroupCard, {
             onClick: () => this.setState({ creatingNewGroup: true })
           }),
+          noGroups && noGroupsMessage,
```

Comment:
you could probably inline this, although I'd put the `isDataLoaded` before the `_.isEmpty(groups)`